deployments/deploy-tests.ps1

```diff
@@ -104,15 +127,14 @@ try{
     #Still need to write the deploy method, but the thought here is that you define everything from options
     #so there is no need for az context or deployment context
 
-    $kvOptions = New-KeyVaultOptions
-
-    $kvOptions.Name="KV-MYAEA-KKZH-ADM-C1-D01" 
-    $kvOptions.SubscriptionName="MYAEA-KKZH-D" 
-    $kvOptions.ResourceGroupName="RG-MYAEA-KKZH-ADM-D"
+    New-PfDeploymentContext
 
-    $kv = New-KeyVaultDefinition -Options $kvOptions
+    $kv = New-PfKeyVaultBuild
+    $kv.Options.Name="KV-MYAEA-KKZH-ADM-C1-D01" 
+    $kv.Options.SubscriptionName="MYAEA-KKZH-D" 
+    $kv.Options.ResourceGroupName="RG-MYAEA-KKZH-ADM-D"
 
-    #$kv.Access.Deploy()
+    Deploy-PfDeploymentContext
```

Comment:
After writing these deploy-tests I am pretty pleased with the direction this is going.  I know we haven't gotten to dependencies, aad security, relationship of resources, but I feel these deploy examples show just how easy it is to deploy with standards or to override the custom ones.  Seems pretty easy now to just deploy 1 resource and seems pretty dynamic to handle a full subscription build out.  So feeling pretty confident we are on the right path here.  Would love to hear your thoughts on these new changes.

Reply:
Yes, some really strong design approach changes in here!  brilliant!  =)